src/tests/ftest/cart/test_group_np_common.h

```diff
@@ -54,13 +73,15 @@ struct test_t test_g = { .t_hold_time = 0,
 			 .t_srv_ctx_num = 1,
 			 .t_roomno = 1082 };
 
-#define CRT_ISEQ_TEST_PING_CHECK /* input fields */		 \
+/* input fields */
+#define CRT_ISEQ_TEST_PING_CHECK				 \
 	((uint32_t)		(age)			CRT_VAR) \
 	((uint32_t)		(days)			CRT_VAR) \
 	((d_string_t)		(name)			CRT_VAR) \
 	((bool)			(bool_val)		CRT_VAR)
 
-#define CRT_OSEQ_TEST_PING_CHECK /* output fields */		 \
+/* output fields */
+#define CRT_OSEQ_TEST_PING_CHECK 				 \
```

Comment:
(style)  please, no space before tabs
(style)  Macros with complex values should be enclosed in parentheses